spec/forms/openid_connect_authorize_form_spec.rb

```diff
@@ -357,23 +357,6 @@
   end
 
   describe '#aal' do
-    context 'when DEFAULT_AAL passed' do
```

Comment:
[question] should there be a default fall-back?

Reply:
Re-added default.

Reply:
But why doesn't it show up in the diff? https://github.com/18F/identity-idp/blob/jcurcio/lg-9178-update-aal-oidc-response/spec/forms/openid_connect_authorize_form_spec.rb#L360

Reply:
Ok, its showing up now.

Reply:
Left a comment above, but I don't think we want a default for the `requested_aal_value`